scripts/update-translations.sh

```diff
@@ -64,11 +64,28 @@ then
     abort
 fi
 
+# Check if OpenAI client is installed
+if ! python -c "import openai" &> /dev/null; then
+    echo "OpenAI client is not installed."
+    echo "Translations will not be updated with LLM."
+    echo "Please install it with: pip install openai"
```

Comment:
We're moving to `uv` from `pip` yeah? Should this be updated?

Reply:
Good catch. Will update it.